Dockerfile

```diff
@@ -30,8 +30,9 @@ COPY --from=build --chown=rsk:rsk /home/rsk/rsk.jar ./
 
 ENV DEFAULT_JVM_OPTS="-Xms4G"
 ENV RSKJ_SYS_PROPS="-Drpc.providers.web.http.bind_address=0.0.0.0 -Drpc.providers.web.http.hosts.0=localhost -Drpc.providers.web.http.hosts.1=127.0.0.1 -Drpc.providers.web.http.hosts.2=::1"
+ENV LOGGING_STDOUT="-Dlogging.stdout=INFO"
```

Comment:
not sure we really need one more env. var... I guess we could reuse `RSKJ_SYS_PROPS`, as indeed this is an rskj specific sys prop, wdyt?

Reply:
I added a new ENV in case an user don’t want to enable logging in the docker container, they can just run the container with the env variable left empty or set it to another LOG_LEVEL. Renamed it to RSKJ_LOG_PROPS